src/extension/src/scripts/rnw-dependencies.ps1

```diff
@@ -0,0 +1,332 @@
+            $msbExeConfig.Save( (GetMsBuild64BitConfigFile) )
+        };
+        Optional = $true
+    },
+    @{
+        # Install the Windows ADK (Assessment and Deployment Kit) to install the wpt (Windows Performance Toolkit) so we can use wpr (Windows Performance Recorder) for performance analysis
+        Name = 'Windows ADK';
+        Tags = @('buildLab');
+        Valid = (Test-Path "${env:ProgramFiles(x86)}\Windows Kits\10\Windows Performance Toolkit\wpr.exe");
+        Install = { choco install -y windows-adk };
+    },
+    @{
+        Name = "React-Native-Windows clone"
+        Tags = @('clone')
+        Valid = try {
+            Test-Path -Path react-native-windows
+            } catch { $false };
+        Install = {
+            & "${env:ProgramFiles}\Git\cmd\git.exe" clone https://github.com/microsoft/react-native-windows.git
+        };
+        Optional = $true
+    }
+);
+
+function IsElevated {
+    return [bool](([System.Security.Principal.WindowsIdentity]::GetCurrent()).groups -match "S-1-5-32-544");
+}
+
+if (!($NoPrompt) -and !(IsElevated)) {
+    Write-Output "rnw-dependencies - this script must run elevated. Exiting.";
+    return;
+}
+
+$NeedsRerun = 0;
+$Installed = 0;
+$filteredRequirements = New-Object System.Collections.Generic.List[object]
+foreach ($req in $requirements)
+{
+    foreach ($tag in $req.Tags)
+    {
+        if ($tagsToInclude.Contains($tag))
+        {
+            $filteredRequirements.Add($req);
+            break;
+        }
+    }
+}
+
+foreach ($req in $filteredRequirements)
+{
+    Write-Host -NoNewline "Checking $($req.Name)    ";
+    if (!($req.Valid)) {
+        if ($req.Optional) {
+            Write-Host -ForegroundColor Yellow " Failed (warn)".PadLeft(50 - $req.Name.Length);
+        }
+        else {
+            Write-Host -ForegroundColor Red " Failed".PadLeft(50 - $req.Name.Length);
+        }
+        if ($req.Install) {
+            if ($Install -or (!$NoPrompt -and (Read-Host "Do you want to install? [y/N]").ToUpperInvariant() -eq 'Y')) {
+                $LASTEXITCODE = 0;
+                $outputFromInstall = Invoke-Command $req.Install -ErrorAction Stop;
+                if ($LASTEXITCODE -ne 0) { throw "Last exit code was non-zero: $LASTEXITCODE - $outputFromInstall"; }
+                else { $Installed++; }
+            } else {
+                $NeedsRerun += !($req.Optional); # don't let failures from optional components fail the script
+            }
+        } else {
+            $NeedsRerun += !($req.Optional);
+        }
+    } else {
+        Write-Host -ForegroundColor Green " OK".PadLeft(50 - $req.Name.Length);
+    }
+}
+
+
+if ($Installed -ne 0) {
+    Write-Output "Installed $Installed dependencies. You may need to close this window for changes to take effect."
+}
+
+if ($NeedsRerun -ne 0) {
+    #TODO: Set comment
```

Comment:
Is this where we said we may add a WTS COMMENT: we commented this to be able to return info blablablah? :)

Reply:
Done.